test/cri-containerd/container_test.go

```diff
@@ -315,6 +316,222 @@ func Test_RunContainer_VirtualDevice_GPU_LCOW(t *testing.T) {
 	}
 }
 
+func Test_RunContainer_VirtualDevice_GPU_Multiple_LCOW(t *testing.T) {
```

Comment:
Need a `requireFeatures(t, featureLCOW)` here.

Reply:
Good point, I think this warrants a new feature flag :)